x/auth/client/rest/requests.go

```diff
@@ -5,5 +5,5 @@ import "github.com/cosmos/cosmos-sdk/x/auth"
 // BroadcastReq requests broadcasting a transaction
 type BroadcastReq struct {
 	Tx     auth.StdTx `json:"tx"`
-	Return string     `json:"return"`
+	Return string     `json:"return"` // TODO: Do we need this?
```

Comment:
cc @alessio

Reply:
? What's that?

Reply:
"return type" as far as I know (async|block|...)

Reply:
It's not used anywhere @faboweb